src/content/docs/browser/new-relic-browser/guides/guide-to-core-web-vitals.mdx

```diff
@@ -0,0 +1,185 @@
+---
+title: Monitor core web vitals
+tags:
+  - Browser
+  - Browser monitoring
+  - Guides
+metaDescription: "A guide to monitor Google's core web vitals."
+---
+
+import browserWebVitalsGif from 'images/browser_screenshot-full_web-vitals-gif.gif'
```

Comment:
I think the '-gif.' mustn't be here. Maybe the macro didn't go well: browser_screenshot-full_web-vitals-gif.gif